logconsumer_test.go

```diff
@@ -323,9 +323,8 @@ func TestContainerLogWithErrClosed(t *testing.T) {
 
 	hitNginx := func() {
 		i, _, err := dind.Exec(ctx, []string{"wget", "--spider", "localhost:" + port.Port()})
-		if err != nil || i > 0 {
-			t.Fatalf("Can't make request to nginx container from dind container")
-		}
+		require.Zerof(t, i, "Can't make request to nginx container from dind container")
```

Comment:
Comment NOT related to this PR, but for the general maintenance of the library:

@stevenh wdyt of keeping the description messages in the checks? Should we keep them, or remove them. We are working in creating more and more contextual error messages, so at some point we'll have much better experience debugging test errors.

Reply:
Personally I don't think they add value in most cases, just adding more for the developer to read. This is particularly the case for error checks. 

The way I look at it is does it make it clearer to the reader why this might have failed, which this type of message I would argue doesn't as its obvious from the line above that the request didn't work.

In this particular example they checks are back to front, error check should come first.